### ISSUE TITLE
Detach Kana

### ISSUE THREAD

**Issue by purepure** [phimage/Erik#52]
Could you detach kana as optional? so someone want to use other parser could work on this too.

Thank

**Comment by phimage** [comment]
good idea. maybe by creating multiple targets, with an ErikCore without Kana
